unifiedWorkspacesView.js

```diff
@@ -2,49 +2,65 @@
 
 //The following is my own version of WorkspacesView.WorkspacesView
 //It is designed to work with only one "Workspace" containing all windows
-
-const { Clutter, Gio, GObject, Meta, Shell, St } = imports.gi;
+const { Clutter, GObject, Meta, Shell, St } = imports.gi;
 const Signals = imports.signals;
 
+// const Tweener = imports.ui.tweener;
 const Main = imports.ui.main;
-const Tweener = imports.ui.tweener;
 const Workspace = imports.ui.workspace;
-
 const WorkspacesView = imports.ui.workspacesView;
+const ExtensionUtils = imports.misc.extensionUtils;
+
+const Me = imports.misc.extensionUtils.getCurrentExtension();
+const UnifiedWorkspace = Me.imports.workspace;
 
 const WORKSPACE_SWITCH_TIME = WorkspacesView.WORKSPACE_SWITCH_TIME;
 
 var UnifiedWorkspacesView = GObject.registerClass(
 class UnifiedWorkspacesView extends WorkspacesView.WorkspacesViewBase {
     _init(monitorIndex, scrollAdjustment) {
+        super._init(monitorIndex);
+        
         let workspaceManager = global.workspace_manager;
 
-        super._init(monitorIndex);
+        for (let functionName of UnifiedWorkspace.replacedFunctions) {
```

Comment:
What's the reasoning for moving this in here instead of in extension.js?

Reply:
I think I just wanted to clean up the code a bit, and as it seemed to be related to this class I moved the code. Now I wonder if `_init` is being actually called.